docker/config/constraints.txt

```diff
@@ -1,17 +1,18 @@
-# downloaded from https://raw.githubusercontent.com/apache/airflow/constraints-1.10.12/constraints-3.7.txt
-# Editable install with no version control (apache-airflow==1.10.12)
-Babel==2.8.0
+# downloaded from https://raw.githubusercontent.com/apache/airflow/constraints-2-0/constraints-3.7.txt
+# Editable install with no version control (apache-airflow==1.10.15)
+Babel==2.9.0
```

Comment:
You need to replace this file with the published constraints file on 1.10.15 branch
https://github.com/apache/airflow/blob/constraints-1-10/constraints-3.7.txt

use that above file as is. i.e. replace the contents of constraints from 1.10.12 branch with the one above.

Reply:
I just need to change the comment?  It is pulled form the constraints of the 1.10.15 branch of Airflow upstream.